packages/block-library/src/post-content/index.php

```diff
@@ -17,6 +17,9 @@ function render_block_core_post_content( $attributes, $content, $block ) {
 	static $seen_ids = array();
 
 	if ( ! isset( $block->context['postId'] ) ) {
+		if ( is_404() ) {
```

Comment:
👋 - I don't think we should be adding hardcoded content like that. Such content should be editable and `removable`. Even if we added a wrapper with a css class here, someone would have to use css to hide it.

In my head this is something to be handled in 404 template but it seems I can't understand how this is not working. For example right now that we have a `404` template in `tt1-blocks` with a custom header block - we can't make this translatable?

Reply:
We don't have any way to translate content in the HTML files for block templates, or block template parts. I've made some other suggestions for how we might work around the problem:
https://github.com/WordPress/gutenberg/pull/33192
https://github.com/WordPress/gutenberg/pull/33217

Reply:
Regardless of the outcome of those PRs (I'm betting on #33217), this feels best absorbed by patterns than by hard-coded server-side logic.

Reply:
Even if the patterns PR merges, I think this is a very useful tool. If I want to create a simple site with only one template, this allows me to do that...

Reply:
> If I want to create a simple site with only one template

I think this is a good aspiration for FSE. Building simple sites should feel as convenient as working with a single template. But I think the current proposal is overloading base FSE blocks with meaning and hardcoded decisions.